lib/load.php

```diff
@@ -71,7 +71,7 @@ function gutenberg_is_experiment_enabled( $name ) {
 require __DIR__ . '/compat/wordpress-6.0/block-editor-settings.php';
 require __DIR__ . '/experimental/block-editor-settings-mobile.php';
 require __DIR__ . '/compat/wordpress-6.0/get-global-styles-and-settings.php';
-require __DIR__ . '/compat/wordpress-5.9/render-svg-filters.php';
```

Comment:
@oandregal Seeing as the new function here completely replaces the old one, can the old file be removed?

Same question for the theme.json file that was updated.

Reply:
Yeah, the old ones are no longer used.

Reply:
Pushed this myself and other minor updates to fast-track this PR.